### DISCUSSION TITLE
动态修改网页标题

### DISCUSSION THREAD

**Discussion by pgygogogo** [vbenjs/vue-vben-admin#2004] [Q&A]
点击路由title就变成了当前路由的标题，请问修改网页title的地方在哪里？

**Comment by kuizuo** [comment]
hook的useTitle中
https://github.com/vbenjs/vue-vben-admin/blob/main/src/hooks/web/useTitle.ts
![image](https://user-images.githubusercontent.com/61005888/175511431-347b43fc-1ee5-4848-b71f-fc1f271e7a5e.png)
